crates/viewer/re_ui/src/ui_ext.rs

```diff
@@ -131,6 +131,21 @@ pub trait UiExt {
         )
     }
 
+    /// Adds a non-interactive, optionally tinted small icon.
+    ///
+    /// Uses [`DesignTokens::small_icon_size`]. Returns the rect where the icon was painted.
+    fn small_icon(&mut self, icon: &Icon, tint: Option<egui::Color32>) -> egui::Rect {
```

Comment:
wouldn't it make more sense to return the `egui::Response`, like for any other widget?

Reply:
Well none is used in the actual implementation as this is not an interactive widget. I could add an explicit `interact()` but it'd add unnecessary complexity imo (id? sense?). User can use `ui.interact()` with the rect if needed.

Reply:
You are right; I thought `allocate_space` returned a response anyways. My bad :)